input/pagecontent/reference.md

```diff
@@ -991,6 +991,13 @@ An extension can have either a value (i.e. a value[x] element) or sub-extensions
 
 Since simple and complex extensions are mutually-exclusive, FSH implementations SHOULD set the value[x] cardinality to 0..0 if sub-extensions are specified, set extension cardinality to 0..0 if constraints are applied to value[x], and signal an error if value[x] and extensions are simultaneously specified.
 
+To indicate that an extension is a [modifier extension](https://hl7.org/fhir/R5/extensibility.html#modifierExtension), authors MUST set the [isModifier](https://hl7.org/fhir/R5/elementdefinition-definitions.html#ElementDefinition.isModifier) flag to `true` on the extension's root element. In addition, the root element's `isModifierReason` must also be provided. This is in accordance with FHIR's [modifier extension requirements](https://hl7.org/fhir/R5/extensibility.html#modifierExtension) and [eld-18](https://hl7.org/fhir/R5/elementdefinition-definitions.html#ElementDefinition.isModifier) invariant. FSH authors can achieve this via rules like the following:
```

Comment:
In this paragraph, you link to the `isModifier` section. Is it also worth linking to https://hl7.org/fhir/R5/elementdefinition-definitions.html#ElementDefinition.isModifierReason when you say "`isModifierReason` must also be provided"?

Reply:
Fixed and rebased into https://github.com/HL7/fhir-shorthand/pull/177/commits/13e5e810ec49d9d27776fef2b831e7ed2172a6c5.